paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -101,6 +89,9 @@ internal class DefaultFlowController @Inject internal constructor(
      */
     lateinit var flowControllerComponent: FlowControllerComponent
 
+    @VisibleForTesting
```

Comment:
Spoke with @ccen-stripe about this one. 

We decided that this was preferred to making StripePaymentLauncher an open class so that it's mockable. The other alternative is mocking the creation of the PaymentLauncher within DefaultFlowController but that seemed complicated. So we're left with making our PaymentLauncher visible for testing.

Reply:
FYI you can use `mock-maker-inline` to mock a final class

Reply:
That saved me ! 

I was able to mock the right payment launcher and set the lifecycle to the correct states to avoid having to make this variable visible. See here: https://github.com/stripe/stripe-android/pull/4175/commits/25ead4abd15e56b4393bd372fb21165f61be11ac

Reply:
🎉 🎉